network/domains-and-dns/reference-content/understanding-dns-records.mdx

```diff
@@ -0,0 +1,142 @@
+---
+meta:
+  title: Understanding DNS records
+  description: This section provides information about the different DNS record types
+content:
+  h1: Understanding DNS records
+  paragraph: This section provides information about the different DNS record types
+tags: domains dns record
+dates:
+  validation: 2023-08-18
+  posted: 2023-08-18
+categories:
+  - network
+---
+
+This page provides information about the different DNS record types available with Domains and DNS and how to use them.
+
+
+Information in DNS zones is categorized and organized through a list of [DNS record](/network/domains-and-dns/concepts/#dns-record) types, called [resource records](/network/domains-and-dns/concepts/#resource-record) (RRs).
+
+Each of these records has a type, an expiration time (Time to Live - TTL) and a name. Certain types of records also have type-specific data.
+
+
+### A record
+
+**A record** stands for **Address record**. It is mostly used to map domain names to the IPv4 address of a specific server. In other words, it allows to connect a domain name to the numerical IP address of a server where the website or online service is hosted.
+
+An A record ensures that when you type a website's name into your browser, your computer knows where to find that website on the internet.
+
+### AAAA record
+
+Just like an A record connects a domain to an IPv4 address, an **AAAA record** does the same thing for the newer IPv6 addresses. It allows devices to locate and communicate with websites and services using the IPv6 protocol.
+
+### ALIAS record
+
+An ALIAS record works exactly like a [CNAME record](#cname-record) but it redirects your domain name to a hostname instead of pointing to an IP address.
```

Comment:
```suggestion
An ALIAS record works exactly like a [CNAME record](#cname-record), but it redirects your domain name to a hostname instead of pointing to an IP address.
```